### ISSUE TITLE
Using EF 6 and EF Core together with same model classes

### ISSUE THREAD

**Issue by burakdobur** [dotnet/efcore#15160]
Hello,

We are building a web application. The project has the structure below:
-QS.Entities (Framework46, Model classes for EF 6)
-QS.DataAccess (Framework46, EF 6 DbContext, and mappings)
-QS.BusinessLayer (Framework46, Repositories, and UoW)
-QS.WebV3 (Framework46, ASP.NET MVC 5)

 The following projects added lately to modernize and port to .NET Core 
-QS.Models (NetStd20, Model classes for EF Core)
-QS.Data (NetCore22, EF Core DbContext, and mappings)
-QS.API (NetCore22, WebAPI project)
-QS.App (NetCore22, Client-side web app, Vue.js)

Last week we tried to merge two model projects to use the same classes across the solution. We generate these new classes through EF Core Power Tools -> Reverse Engineer command. Which generates classes for our database. 

When replaced all the classes with newly generated models we realized that EF Core's many-to-many navigations are slightly different than EF 6's. For example, we have the following tables, Task, Files, and TaskFiles.

EF 6 uses Task class's navigation like this;
public ICollection<File> Files { get; set; }
In our BussinessLayer this navigation property is used to modify relation. 

But In the newly generated model's navigation are like this:
public ICollection<TaskFile> TaskFiles { get; set; }
And this navigation should be used to modify join-table directly to change relation. 

In EF Core's blog and GitHub discussions we've seen that in preview builds of EF Core, navigation properties would work without join entity as in EF 6. But there is no example of how to configure mappings in these pages. 

In short, we want to use the same model classes for EF 6 and EF Core to merge the following projects:
-QuaServices.Entities (Model classes for EF 6)
-QuaServices.DataAccess (EF 6 DbContext and mappings)
and
-QuaServices.Models (Model classes for EF Core)
-QuaServices.Data (EF Core DbContext and mappings) 

Does anybody know how to configure many-to-many without join entity in EF Core?

**Comment by ajcvickers** [comment]
See #1368
Duplicate of #10508

**Comment by burakdobur** [comment]
I think this is not a duplicate issue. The attached conversations are about how this should be implemented or implemented at all. My question is about the missing documentation and usage guide.

**Comment by ajcvickers** [comment]
@burakdobur What specifically is missing?

**Comment by burakdobur** [comment]
I can't find any information about how to configure many-to-many relationship -without join entity- using property bag navigations.

**Comment by ajcvickers** [comment]
@burakdobur That's because it is not yet implemented. The duplicate issue is still open indicating that it is not yet implemented.
